pkg/api/plugins.go

```diff
@@ -456,6 +456,8 @@ func (hs *HTTPServer) InstallPlugin(c *contextmodel.ReqContext) response.Respons
 	}
 	pluginID := web.Params(c.Req)[":pluginId"]
 
+	hs.log.Info(fmt.Sprintf("Plugin %s install/update requested", pluginID), "user", c.Login)
```

Comment:
I think we should use structured logging also for the plugin ID:

```suggestion
	hs.log.Info("Plugin install/update requested", "pluginID", pluginID, "user", c.Login)
```

This would allow us to more easily query for the plugin ID in Loki using the `logfmt` stage.

This would also make it consistent with how everything else is logged in Grafana core.